### PR TITLE
added clone factory

### PR DESCRIPTION
**Pull request by BritikovKI**
Added clone factory to the clone factory.

### CODE REVIEW

File: contracts/schemes/FarmFactory.sol [hunk 1]
```diff
@@ -32,6 +33,7 @@ contract FarmFactory {
 
 
 	Avatar public avatar;
+	StakingRewards public father;
```
Comment (kinspotting) [inline contracts/schemes/FarmFactory.sol:36]: pls change `father` to `parent`

File: contracts/schemes/FarmFactory.sol [hunk 2]
```diff
@@ -57,6 +59,18 @@ contract FarmFactory {
 	function initialize(Avatar _avatar) external initializer {
 		require(_avatar != Avatar(0), 			"FarmFactory: avatar cannot be null");
 		avatar = _avatar;
+		//		TODO: there is a possibility to make it more flexible, and to send father address as a constructor parameter
```
Comment (kinspotting) [inline contracts/schemes/FarmFactory.sol:62]: let's do it on construction

**Review by kinspotting — CHANGES_REQUESTED** [review]
needs minor changes

File: contracts/schemes/FarmFactory.sol (hunk 1 above)
Comment (BritikovKI) [reply contracts/schemes/FarmFactory.sol]: Done

File: contracts/schemes/FarmFactory.sol (hunk 2 above)
Comment (BritikovKI) [reply contracts/schemes/FarmFactory.sol]: Done

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**